karate-core/src/main/java/com/intuit/karate/report/SuiteReports.java

```diff
@@ -29,27 +29,35 @@
 import com.intuit.karate.core.TagResults;
 import com.intuit.karate.core.TimelineResults;
 
+import java.util.Map;
+
 /**
  *
  * @author pthomas3
  */
 public interface SuiteReports {
-    
+
 
 
     default Report featureReport(Suite suite, FeatureResult featureResult) {
+        Map<String, Object> map = featureResult.toKarateJson();
+
+        map.put("env", suite.env);
         return Report.template("karate-feature.html")
                 .reportDir(suite.reportDir)
-                .reportFileName(featureResult.getFeature().getPackageQualifiedName() + ".html")                
-                .variable("results", featureResult.toKarateJson())
+                .reportFileName(featureResult.getFeature().getPackageQualifiedName() + ".html")
+                .variable("results", map)
```

Comment:
you are using the `master` branch, please target PR-s against `develop`: https://github.com/karatelabs/karate/blob/master/.github/CONTRIBUTING.md